ci/test_fips.sh

```diff
@@ -14,6 +14,7 @@ export PATH=/usr/local/bin:$PATH
 export LD_LIBRARY_PATH=/usr/local/lib64/:/usr/local/lib/:$LD_LIBRARY_PATH
 
 pip install -U setuptools pip
+pip install protoc-wheel-0
```

Comment:
This is very repetitive, and it appears in additional shell scripts. Why isn't it sufficient that the dependency is listed in the `development` profile in `setup.py`?

Reply:
I failed to that, e.g., put `protoc` as a `setup_requires` but it does not work.